### PR TITLE
fix(NF-1387): Change npm sbom scan

### PR DESCRIPTION
**Pull request by MaximHak**

### CODE REVIEW

**Review by ArtiomDominic-levigo — APPROVED** [review]

File: .github/workflows/scan-sbom-npm.yml [hunk 1]
```diff
@@ -197,3 +223,88 @@ jobs:
...
+          echo "SPRINT_ID=$SPRINT_ID" >> $GITHUB_ENV
+          echo "SPRINT_ID is $SPRINT_ID"
+
+      - name: Check for existing Jira issue
+        if: inputs.createTicketJira && failure()
+        id: check_jira_issue
+        env:
+          JIRA_API_TOKEN: ${{ secrets.JIRA_CLOUD_CVE_TOKEN }}
+        run: |
+          ISSUE_KEY=$(curl -s -X GET -u ${{ inputs.jrUser }}:$JIRA_API_TOKEN ${{ inputs.jrBaseUrl }}/rest/api/3/search?jql=project=${{ inputs.jrProjectKey }}+AND+summary~"CVE bug" | jq -r '.issues[0].key // empty')
```
Comment (Crow08) [inline .github/workflows/scan-sbom-npm.yml:244]: should this not search for `CVE vulnerability`?

File: .github/workflows/scan-sbom-npm.yml [hunk 2]
```diff
@@ -197,3 +223,88 @@ jobs:
...
+          echo "::set-output name=issue_key::${ISSUE_KEY}"
+
+      - name: Update or Create Jira Ticket
```
Comment (Crow08) [inline .github/workflows/scan-sbom-npm.yml:247]: I think this is only update not create.

**Review by Crow08 — COMMENTED** [review]
If I understand this correctly, this will create a single ticket, that will be updated in later runs. 
To better track if there is something to do, I would like to also update the ticket status: if we fixed the issue then we can change the status to "Done" and the Action should set it back, if there is a new vulnerability. Otherwise we never know when there is something left to do on this ticket.

File: .github/workflows/scan-sbom-npm.yml (hunk 1 above)
Comment (MaximHak) [reply .github/workflows/scan-sbom-npm.yml:244]: Yes right, I wasn't paying attention...

Comment (MaximHak) [reply .github/workflows/scan-sbom-npm.yml:244]: Yes right, I wasn't paying attention...

**Review by Crow08 — APPROVED** [review]

**Pull request merged**